Automattic/jetpack
# Social Previews: move block to a plugin

Fixes #21061

## Proposed changes:

Since this is really an extension plugin instead of a block, let's register it like a plugin.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Posts > Add New
* Add some content, including a featured image. Save your draft.
* Open the Jetpack plugin
* Open the Social Previews panel.
* Make sure it still works.
